realtime/shake_event.py

```diff
@@ -803,18 +803,13 @@ def sorted_impacted_cities(self, row_count=5):
             cities.append(city)
         LOGGER.debug('%s features added to sorted impacted cities list.')
         # LOGGER.exception(cities)
-        sorted_cities = sorted(cities,
-                               key=lambda d: (
-                               # we want to use whole no's for sort
-                               - d['mmi-int'],
-                               - d['population'],
-                               d['name'],
-                               d['mmi'],  # not decimals
-                               d['roman'],
-                               d['dist_to'],
-                               d['dir_to'],
-                               d['dir_from'],
-                               d['id']))
+        sorted_cities = sorted(
+            cities,
+            key=lambda d: (
+                # we want to use whole no's for sort
+                -d['mmi-int'], -d['population'], d['name'],
```

Comment:
can you keep this one item per line as per our 'pull left' policy (if items dont fit on one line they should be placed one item per line)